recipes/cuda-histogram/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "cuda_histogram" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a3672c474dd094e2fefd364757d8fb6aa016af6e2ffee356e2a9a78c88f58690
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

This just ensures that all the build backend dependencies are properly getting installed through conda-build on `host`.

---

Thanks!